packages/lesmis-server/.env.example

```diff
@@ -3,19 +3,7 @@ PORT=
 API_CLIENT_NAME=
 API_CLIENT_PASSWORD=
 API_CLIENT_SALT=
-
-DB_NAME=
-DB_PASSWORD=
-DB_URL=
-DB_USER=
+ 
 
 MICROSERVICE_CLIENT_SECRET=
 MICROSERVICE_CLIENT_USERNAME=
```

Comment:
These variables are meant to have the exact same values as `API_CLIENT_NAME` and `API_CLIENT_PASSWORD`.

I think I was meant to clean them up as part of a recent change but missed them. Can we remove these and replace the one instance they're being used with `API_CLIENT_NAME` and `API_CLIENT_PASSWORD`? :pray: